src/main/java/it/pagopa/interop/probing/scheduler/scheduler/ScheduledTasks.java

```diff
@@ -40,22 +44,29 @@ public void scheduleFixedDelayTask() {
     logger.logSchedulerStart();
     try {
       Integer offset = 0;
+      PollingEserviceResponse response = PollingEserviceResponse.builder().build();
```

Comment:
does this init have a purpose considering that `response` value is replaced right away?

Reply:
Done